src/test/java/org/beanpod/switchboard/dao/StreamLogDaoImplTest.java

```diff
@@ -41,4 +47,14 @@ final void createStreamLogTest() {
 
     assertEquals(streamLogDto, actualStreamLog);
   }
+
+  @Test
+  final void getStreamLogsTest() {
+    when(logStreamMapper.toStreamLogModels(any())).thenReturn(streamLogModelList);
+    when(logStreamRepository.findByStreamId("1")).thenReturn(streamLogList);
+
+    List<StreamLogModel> listOfStreamLogModels_actual = streamLogDao.getStreamLogs((long) 1);
```

Comment:
Isn't there a way to do it without casting? Like `1L` instead of `(long) 1`? Or does that not work?

Reply:
Thanks for the suggestion. I forgot that we can do something like #L to represent long number :)! I have updated my tests accordingly.